libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.scss

```diff
@@ -0,0 +1,10 @@
+@import "~fundamental-styles/dist/resizable-card-layout";
+
+.fd-resizable-card-expansion {
+    transition-duration: 350ms;
+    animation-timing-function: ease-in-out;
+}
+
+.fd-resizable-card-border-display {
+    display: flex;
+}
```

Comment:
does not belong to fund-ngx

Reply:
currently i have not got animation duration values from designer. I had asked them about couple of weeks ago some other questions, but this component is not in their priority list. and they are not able to provide the answer.
I have asked them again today. whenever answer comes, we can move these values to styles.
hope this is fine to unlock the development. i will keep an issue open for this.